newIDE/app/public/external/jfxr/jfxr-main.js

```diff
@@ -71,18 +71,29 @@ ipcRenderer.on('jfxr-open', (event, receivedOptions) => {
     projectPath: receivedOptions.projectPath,
     initialResourcePath: receivedOptions.resourcePath,
     extension: '.wav',
-    headerStyle
   });
+
+  electronWindow.setTitle(
+    'GDevelop Sound Effects Editor - ' +
```

Comment:
Not sure if we should "hide" the name of the editor we're embedding. What do you think of at least keeping their original names like this: "Sound Effects Editor (Jfxr) - "?

Reply:
You're right - I need to re-add the names. That way the any editor specific bugs will be more likely to be submitted to the editor's github, not gdevelop's. We shouldn't mislead the users in any way that these editors are a part of GD